### ISSUE TITLE
Is translations possible using only route attributes?

### ISSUE THREAD

**Issue by vallieresch** [Dresel/RouteLocalization#77]
As I am using a custom scaffolder, it would be much easier if the translations would be replaced by a route attribute on the controllers/actions. 

Something like: 
[LocalizedRoute("{culture}/services/contact-us", "en-CA")]
[LocalizedRoute("{culture}/services/nous-joindre", "fr-CA")]

Is this something that is already supported? If not, is there another way around to meet my goal and can this feature be added?

**Comment by Dresel** [comment]
It's not supported out of the box, but maybe you could do something like [this](https://github.com/Dresel/RouteLocalization/issues/46#issuecomment-101253238)?

**Comment by vallieresch** [comment]
Hmmm I fail to see how I could define 2 localized routes for an action using that method. I can't do a redirect.

**Comment by Dresel** [comment]
See this [localized-route-attribute](https://github.com/Dresel/RouteLocalization/tree/localized-route-attribute) branch and the [RouteConfig.cs](https://github.com/Dresel/RouteLocalization/blob/localized-route-attribute/Source/RouteLocalization.Mvc.Sample/App_Start/RouteConfig.cs#L59) and [HomeController.cs](https://github.com/Dresel/RouteLocalization/blob/localized-route-attribute/Source/RouteLocalization.Mvc.Sample/Controllers/HomeController.cs#L10) of the sample project.

**Comment by vallieresch** [comment]
Thanks Dresel! It's working exactly as expected. The only thing I cannot get working is the CurrentCulture and CurrentUICulture to switch automatically. I tried both of your CultureSensitiveActionFilterAttribute and CultureSensitiveHttpModule but no change.

**Comment by Dresel** [comment]
Does it work in the sample project of the localized-route-attribute? "en/Welcome" should switch to englisch culture and "de/Willkommen" to german culture. Maybe you have something else registered that overrides the filter / module.

**Comment by vallieresch** [comment]
Yes your branch works. I got it working on my test project. My LocalizedRouteAttribute was inheriting from RouteFactoryAttribute. I also didn't have the [Route("Index")]. Anyway to drop the [Route] attribute altogether? I imagine this serves as the default route that is then translated by the LocalizedRoute attribute.
